pkg/template/servicebroker/catalog_test.go

```diff
@@ -51,7 +50,7 @@ func TestServiceFromTemplate(t *testing.T) {
 	expectedService := &api.Service{
 		Name:        "name",
 		ID:          "ee33151d-a34d-442d-a0ca-6353b73a58fd",
-		Description: "description",
+		Description: noDescriptionProvided,
```

Comment:
should probably have both tests.  we wouldn't want to break the actual description mapping at some point.